bip-0009.mediawiki

```diff
@@ -24,6 +24,7 @@ In addition, BIP 34 made the integer comparison (nVersion >= 2) a consensus rule
 
 Each soft fork deployment is specified by the following per-chain parameters (further elaborated below):
 
+# The '''name''' specifies a very brief description of the soft fork, reasonable for use as an identifier.
```

Comment:
I would suggest the for most of the time, most softforks will role out one consensus BIP so the sensible name would be `bipxxxx` unless you have a case like with the bip68+112+113 softfork, in which case we labelled that `csv`. Therefore I would suggest default of `bipxxxx` unless it's a bundle.

Thinking more, we probably need a table of soft forks somewhere with name, bip(s), starttime, timeout.